resources/views/backend/categories/create.blade.php

```diff
@@ -0,0 +1,94 @@
+@extends('backend.layouts.master')
+
+@section('title', trans('labels.create_cate'))
+
+@section('content')
+    <div id="page-wrapper">
+        <div class="row">
+            <div class="col-lg-12">
+                <h1 class="page-header">{!! trans('labels.add_cate') !!}</h1>
+            </div>
+            <!-- /.col-lg-12 -->
+        </div>
+        <!-- /.row --> 
+        <div class="row">
+            <div class="col-lg-12">
+                <div class="panel panel-default">
+                    @if (isset($errors) && $errors->any())
+                        <div class="alert alert-danger">
+                            {!! trans('labels.prolem_input') !!}
+                            <ul>
+                                @foreach ($errors->all() as $error)
+                                    <li>{{ $error }}</li>
+                                @endforeach
+                            </ul>
+                        </div>
+                    @endif
+                    <div class="panel-body">
+                        <div class="row">
+                            <div class="col-lg-12">
+                                <form role="form" action="{{ url('admin/categories') }}" method="post" enctype="multipart/form-data">
+                                    {{ csrf_field() }}
+                                    <div class="form-group">
+                                       
+                                        <label>Category Parent</label>
+                                        <select name="parent_id" class="form-control">
```

Comment:
this code is not good, so many `foreach` loop. If you want to select the `parent_id`, just list all category that available in DB.